### ISSUE TITLE
miss the edge gradient loss in HairMattingLoss

### ISSUE THREAD

**Issue by VincentCheng1995** [YBIGTA/pytorch-hair-segmentation#31]
https://github.com/YBIGTA/pytorch-hair-segmentation/blob/587db78b6b1c2188fee866897a0fe245c058b5bb/networks/mobile_hair.py#L202
Hi, thank you for your great contribution! 
But, do you forget to add loss2 to the final loss?
Which I think the loss should be:
`loss = (1-self.ratio_of_gradient)*self.bce_loss(pred, true) + loss2*self.ratio_of_gradient.`

**Comment by davinnovation** [comment]
Thx for interest our project. Our HairMattingLoss for calculating "mask-image gradient consistency loss" part is quite buggy, so we didn't use it temporarily. 
Add option for your suggestion in [this PR](https://github.com/YBIGTA/pytorch-hair-segmentation/commit/4146dfdf7382168381d937c52b8c71dc304b71c7)
